src/components/ui/toggle.tsx

```
@@ -0,0 +1,45 @@
+"use client"
+
```

ESLint

[error] Missing semicolon.

(semi)